hamster_gtk/misc/widgets/raw_fact_entry.py

```diff
@@ -95,8 +132,17 @@ def remove_prefix(segment, string):
                 result = string[1:]
             return result
 
-        result = self.match.group(self.current_segment)
-        result = remove_prefix(self.current_segment, result)
+        if self.current_segment is None:
+            return
+
+        if self.current_segment == 'activity+category':
+            result = self.match.group('activity')
```

Comment:
To allow matching just on categories, something like `result = (self.match.group('activity') or '') + (self.match.group('category') or '')` should be used.